benchmarks/mwt/setup/00-kudo-check/00-assert.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: kudo-manager
+  name: kudo-controller-manager-0
+  namespace: kudo-system
+status:
+  containerStatuses:
+  - name: manager
+    ready: true
+    image: docker.io/kudobuilder/controller:v0.14.0-rc1
```

Comment:
can we template versions somehow? 🤔 this always changes between MWTs and same goes for `operatorversion` `cassandra-0.1.2`

Reply:
I debated this... we can just not include the container image and we will be good.   It depends on how exacting we want to be ...   I'll remove it for now